src/actions/post-photo.ts

```
@@ -0,0 +1,42 @@
+"use server";
+
+import { PHOTO_POST } from "@/functions/api";
+import { apiError } from "@/functions/api-error";
+import { ActionResponse } from "@/shared/types";
+import { revalidateTag } from "next/cache";
+import { cookies } from "next/headers";
+import { redirect, RedirectType } from "next/navigation";
+
+export async function postPhoto(
+  _state: {},
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)